warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/TableSplitsCache.java

```diff
@@ -208,19 +249,26 @@ private boolean exceedsMaxSplitsDeviation(Map<String,Integer> tmpSplitsPerTable)
     @Override
     protected void readCache(BufferedReader in, String delimiter) throws IOException {
         this.splits = new HashMap<>();
+        this.splitLocations = new HashMap<>();
         String line;
         String tableName = null;
         List<Text> splits = null;
+        SortedMap<Text,String> tmpSplitLocations = null;
         while ((line = in.readLine()) != null) {
             String[] parts = StringUtils.split(line, delimiter);
             if (tableName == null || !tableName.equals(parts[0])) {
                 tableName = parts[0];
                 splits = new ArrayList<>();
+                tmpSplitLocations = new TreeMap<>();
                 this.splits.put(tableName, splits);
+                this.splitLocations.put(tableName, tmpSplitLocations);
             }
```

Comment:
if the tmpSplitLocations is empty, should we clear it out of the splitLocations map?